Include/internal/pycore_pystate.h

```diff
@@ -30,6 +30,17 @@ _Py_IsMainInterpreter(PyInterpreterState *interp)
 }
 
 
+static inline const PyConfig *
+_Py_GetMainConfig(void)
+{
+    PyInterpreterState *interp = _PyRuntime.interpreters.main;
+    if (interp == NULL) {
+        return NULL;
+    }
+    return _PyInterpreterState_GetConfig(interp);
+}
+
+
```

Comment:
Is this still needed? I can't find any uses of it.

Reply:
I missed it when I was cleaning out all that running-in-the-source-tree logic.  I'll probably add it back in later, but it doesn't need to be in this PR.